pregrasp_offset: 0.15
ee_offset: 0.15
init_pose: []
basket_pregrasp: []